docs/source/user/api.rst

```diff
@@ -510,6 +571,11 @@ anyway but using the HTTP status code 401, this way consumers can recognize
 these users and trigger the appropriate response needed (eg: reject them
 or initiate account verification).
 
+If an existing user account tries to log in with the URL for a different organization,
+then the user is also added as a member of the new organization.
+Please refer to the `"Registering to Multiple Organizations" <#registering-to-multiple-organizations>`_
+section of this documentation for details.
```

Comment:
I would rephrase this: different organization doesn't make sense because different to what?

I would talk about trying to authenticate to an organization of which they're not member of and in that case they would be automatically added as members (if registration is enabled for that org).

```
Please refer to the `"Registering to Multiple Organizations" <#registering-to-multiple-organizations>`_
section of this documentation for details.
```

I suggest shorten it:
```
Please refer to `"Registering to Multiple Organizations" <#registering-to-multiple-organizations>`_.```